docs/source/cpp/flight.rst

```diff
@@ -172,6 +172,110 @@ request/response. On the server, they can inspect incoming headers and
 fail the request; hence, they can be used to implement custom
 authentication methods.
 
+.. _flight-best-practices:
+
+Best practices
```

Comment:
Link to the grpc best practices doc as well?

Reply:
They are linked below as `best gRPC practices`